README.md

```diff
@@ -164,8 +164,6 @@ Differences are noted here:
 |`commandTimeouts`|Custom timeout(s) in milliseconds for WDA backend commands execution. This might be useful if WDA backend freezes unexpectedly or requires too much time to fail and blocks automated test execution. The value is expected to be of type string and can either contain max milliseconds to wait for each WDA command to be executed before terminating the session forcefully or a valid JSON string, where keys are internal Appium command names (you can find these in logs, look for "Executing command 'command_name'" records) and values are timeouts in milliseconds. You can also set the 'default' key to assign the timeout for all other commands not explicitly enumerated as JSON keys.|`'120000'`, `'{"findElement": 40000, "findElements": 40000, "setValue": 20000, "default": 120000}'`|
 |`maxTypingFrequency`|Maximum frequency of keystrokes for typing and clear. If your tests are failing because of typing errors, you may want to adjust this. Defaults to 60 keystrokes per minute.|e.g., `30`|
 |`simpleIsVisibleCheck`|Use native methods for determining visibility of elements. In some cases this takes a long time. Setting this capability to `false` will cause the system to use the position and size of elements to make sure they are visible on the screen. This can, however, lead to false results in some situations. Defaults to `false`, except iOS 9.3, where it defaults to `true`.|e.g., `true`, `false`|
-|`startIWDP`|Set this to `true` if you want to start ios_webkit_debug proxy server automatically for accessing webviews on iOS. The capatibility only works for real device automation. Defaults to `false`.|e.g., `true`|
-|`webkitDebugProxyPort`|Local port number used for communication with ios-webkit-debug-proxy. Only relevant for real devices. The default value equals to `27753`.|e.g. `20000`|
```

Comment:
Don'r we need this setting anymore? Are the ports assigned automatically when parallel testing sessions connect to web views @umutuzgur ?

Reply:
@mykola-mokhnach There are not server ports anymore. `appium-remote-debugger` talks to the phone directly over `usbmuxd`

Reply:
good, then the cap should be removed from desired caps and other places where it is used as well

Reply:
Yea, we should find them in appium doc like https://github.com/appium/appium/blob/4fb04006157e56656324863f22f0d6501aad08f8/docs/en/advanced-concepts/parallel-tests.md and appium-ios-driver.
(in this repo, webkitDebugProxyPort was just passed to ios-driver'iwdp)